File: cpp/WCFExampleCPP/WCFExampleCPP/IEvalService.cpp

// needed since this AIN'T C#!!
#include "Eval.cpp"

using namespace System;
using namespace System::Collections::Generic;
using namespace System::ServiceModel;

namespace WCFExampleCPP {

	[ServiceContract]
	public interface class IEvalService
	{
	public:
		[OperationContract]
		virtual void SubmitEval(Eval^ eval);
		[OperationContract]
		virtual List<Eval^>^ GetEvals();
		[OperationContract]
		virtual void RemoveEval(String^ id);
	};

}

File: csharp/WCFEvalExample/Backup/WCFEvalExample/IEvalService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ServiceModel;

namespace WCFEvalExample
{
    [ServiceContract]
    public interface IEvalService
    {
        [OperationContract]
        void SubmitEval(Eval eval);
        [OperationContract]
        List<Eval> GetEvals();
        [OperationContract]
        void RemoveEval(string id);
    }
}
